src/server/pages/error.js

```diff
@@ -59,6 +61,8 @@ export default function error ({ baseUrl, basePath, error = 'default', res }) {
   return (
     <div className='error'>
       <h1>{heading}</h1>
+      {locale &&
+        <p>Locale: {locale || 'N/A'}</p>}
```

Comment:
No use for the `locale &&` check if you fall back to `N/A` anyway
```suggestion
      <p>Locale: {locale || 'N/A'}</p>
```